util/Sequencer/Sequencer.gd

```diff
@@ -58,4 +57,5 @@ func seek(sec: float):
 	pass
 
 func finished(_anim):
-	Global.emit_signal("finished")
+	pass
+#	Global.emit_signal("finished")
```

Comment:
```suggestion
```
this comment can be removed and a signal to sequecer can be added that will be emitted